### PR TITLE
kubetest2-kops: rename control-plane-size flag to control-plane-count

### PR DESCRIPTION
**Pull request by justinsb**
- kubetest2-kops: rename control-plane-size flag to control-plane-count
- test scenarios: recognize KOPS_CONTROL_PLANE_COUNT

### CODE REVIEW

**Comment by justinsb** [comment]
Issue #15559

**Comment by rifelpet** [comment]
We might need to coordinate updates in other repos that use kubetest2-kops to also use this new flag. Maybe we recognize both as an intermediate step?

https://github.com/search?type=code&q=org%3Akubernetes+control-plane-size

**Comment by johngmyers** [comment]
We should use cobra's `SetNormalizeFunc` to accept both flag names for a transition period.

**Comment by justinsb** [comment]
I forgot about cloud-provider-gce - thanks @rifelpet 

Also figured out how to plug in the SetNormalizeFunc - thanks @johngmyers

File: tests/e2e/kubetest2-kops/deployer/deployer.go [hunk 1]
```diff
@@ -48,14 +48,16 @@ type deployer struct {
...
+	ClusterName    string   `flag:"cluster-name" desc:"The FQDN to use for the cluster name"`
+	CloudProvider  string   `flag:"cloud-provider" desc:"Which cloud provider to use"`
+	GCPProject     string   `flag:"gcp-project" desc:"Which GCP Project to use when --cloud-provider=gce"`
+	Env            []string `flag:"env" desc:"Additional env vars to set for kops commands in NAME=VALUE format"`
+	CreateArgs     string   `flag:"create-args" desc:"Extra space-separated arguments passed to 'kops create cluster'"`
+	KopsBinaryPath string   `flag:"kops-binary-path" desc:"The path to kops executable used for testing"`
+	createBucket   bool     `flag:"-"`
+
+	// ControlPlaneCount specifies the number of VMs in the control-plane.
+	ControlPlaneCount int `flag:"control-plane-count" desc:"Number of control plane instances"`
```
Comment (johngmyers) [inline tests/e2e/kubetest2-kops/deployer/deployer.go:60]: Hyphenate compound modifiers.

```suggestion
	ControlPlaneCount int `flag:"control-plane-count" desc:"Number of control-plane instances"`
```

**Review by johngmyers — COMMENTED** [review]
Grammar nit

**Review by hakman — APPROVED** [review]

**Comment by johngmyers** [comment]
This broke a bunch of upgrade tests because old versions don't support the new flag.

https://prow.k8s.io/view/gs/kubernetes-jenkins/logs/e2e-kops-aws-upgrade-k121-ko125-to-k122-ko127/1682382988182032384

**Pull request merged**